src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+  height: 60px;
+}
+
+.nav__list {
+  padding: 0;
+}
+
+img {
+  margin: 40px;
+}
+
+.nav {
+  margin: 0 50px;
+  /* padding: 0 10px; */
+}
+
+.nav__item {
+  display: inline-block;
+  position: relative;
+  padding-left: 20px;
+}
+
+.nav__link {
+  --height: 60px;
+  height: var(--height);
+  line-height: var(--height);
+  display: inline-block;
+  font-size: 12px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  color: rgba(0, 0, 0, 1);
+  text-transform: uppercase;
+}
+
+a {
+  text-decoration: none;
+}
```

Comment:
do not need this, instead
you need to remove list stylling form the list by`
list-style: none;` for `nav__list`

Reply:
as I have mentioned above I tried to do so but it didn't affect the 'a links'